app/config.py

```diff
@@ -133,6 +134,9 @@ class Config(object):
     # PII check
     SCAN_FOR_PII = os.getenv("SCAN_FOR_PII", False)
 
+    # Documentation and Translation
+    DOCUMENTATION_DOMAIN = os.getenv('DOCUMENTATION_DOMAIN', f'https://documentation.{ADMIN_BASE_DOMAIN}')
```

Comment:
Can you leave out the `https://` from here? The `_DOMAIN` name gives the impression that it will not include a protocol.

The same variable/env name on the admin does not include the protocol https://github.com/cds-snc/notification-admin/blob/f947e0d2053b372d2321670dcf7d0260287dba8d/app/config.py#L32

If it's a burden to add the protocol multiple maybe you can create a small helper to add the protocol and the `/en/` prefix if you feel like it

Reply:
agreed, refactored and added a helper method called `get_document_url` that follows other utility functions